client/components/PubEdgeListing/pubEdgeListingCard.scss

```diff
@@ -5,18 +5,25 @@
 	border-width: 1px;
 	margin-bottom: 24px;
 	position: relative;
-	transition: filter 100ms linear;
+	transition: all 100ms linear;
 
 	&.in-pub-body {
-		filter: grayscale(100%);
+		> * {
+			filter: grayscale(100%);
+		}
```

Comment:
Instead of making the whole card grayscale, make all children _except_ the `.relation` element (with the return arrow icon) grayscale.